hydra_python_core/doc_maker.py

```diff
@@ -2,9 +2,9 @@
 create a HydraDoc object for it'''
 import re
 import json
-from hydrus.samples.doc_writer_sample import api_doc as sample_document
-from hydrus.hydraspec.doc_writer import HydraDoc, HydraClass, HydraClassProp, HydraClassOp
-from hydrus.hydraspec.doc_writer import HydraStatus
+from samples.doc_writer_sample import api_doc as sample_document
```

Comment:
Don't think this will work as samples is not a python module

Reply:
@ShravanDoda why you would ever want to import a sample in one of the modules? Samples are there as examples, not to be used in the code.

Reply:
@xadahiya @Mec-iS 
`doc_maker` uses `doc_writer_sample` when it's being run directly and not being imported.
<pre>if __name__ == "__main__":
    api_doc = create_doc(sample_document.generate())
    print(json.dumps(api_doc.generate(), indent=4, sort_keys=True))</pre>

Reply:
This is the way it's being done in `hydraspec`

Reply:
then `doc_maker` should be implemented as library module only and that `__main__` block removed.
If we want to keep that in the package, it should be moved into a `main.py` or `cli.py` module.

Reply:
@xadahiya
`doc_writer_sample` is a python module. It uses `doc_writer` to create a `doc_writer_sample_output.py`.

Reply:
@Mec-iS shall I add a commit for removing the `__main__` block?

Reply:
this PR has been closed and moved. See comment below.